### PR TITLE
Print some more helpful debug messages

### PR DESCRIPTION
**Pull request by swissspidy**
See #46.

### CODE REVIEW

File: src/MakePotCommand.php [hunk 1]
```diff
@@ -300,6 +319,8 @@ protected function makepot() {
 
 		// Add existing strings first but don't keep headers.
 		if ( $this->merge ) {
+			WP_CLI::debug( sprintf( 'Mergining with existing POT file: %s', $this->merge ), 'make-pot' );
```
Comment (schlessera) [inline src/MakePotCommand.php]: There's a typo here: `Mergining` => `Merging`

File: src/MakePotCommand.php [hunk 2]
```diff
@@ -147,38 +151,50 @@ public function handle_arguments( $args, $assoc_args ) {
...
-					WP_CLI::error( sprintf( 'Invalid file %s', $assoc_args['merge'] ) );
-				}
-
 				$this->merge = $assoc_args['merge'];
 			}
-		}
 
-		WP_CLI::debug( sprintf( 'Destination: %s', $this->destination ), 'make-pot' );
+			if ( isset( $this->merge ) && ! file_exists( $this->merge ) ) {
+				WP_CLI::warning( sprintf( 'POT file to merge with does not exist: %s', $this->merge ) );
```
Comment (schlessera) [inline src/MakePotCommand.php]: Typo: `with` => `which`

File: src/MakePotCommand.php [hunk 3]
```diff
@@ -377,7 +398,17 @@ protected function makepot() {
 			}
 		}
 
-		return PotGenerator::toFile( $this->translations, $this->destination );
+		$result = PotGenerator::toFile( $this->translations, $this->destination );
+
+		$translations_count = count( $this->translations );
+
+		if ( 1 === $translations_count ) {
+			WP_CLI::debug( sprintf( 'Extracted %d string', count( $this->translations ) ), 'make-pot' );
```
Comment (schlessera) [inline src/MakePotCommand.php]: Reuse `$translations_count` here

File: src/MakePotCommand.php [hunk 4]
```diff
@@ -377,7 +398,17 @@ protected function makepot() {
...
+		} else {
+			WP_CLI::debug( sprintf( 'Extracted %d strings', count( $this->translations ) ), 'make-pot' );
```
Comment (schlessera) [inline src/MakePotCommand.php]: Reuse `$translations_count` here

File: src/MakePotCommand.php (hunk 2 above)
Comment (schlessera) [reply src/MakePotCommand.php]: Ah, I read it in a different way: "Here is a reference to the POT file we wanted to merge but _which_ does not exist: ...".

Some random ideas for an alternative...

* "Merge target does not exist: ..."
* "POT file to be merged does not exist: ..."
* "Invalid POT file set as merge target: ..."
* "File to merge is not valid: ..."
* "Invalid file provided to --merge: ..."

File: src/MakePotCommand.php (hunk 4 above)
Comment (schlessera) [reply src/MakePotCommand.php]: Maybe we should move over the `Inflector` class from `wp-cli/scaffold-package` for strings like this.
You could then do something along these lines:
```
WP_CLI::debug( sprintf(
   'Extracted %d %s',
   $translations_count,
   Utils\maybe_pluralize( 'string', $translations_count )
), 'make-pot' );
```

Comment (schlessera) [reply src/MakePotCommand.php]: See https://github.com/wp-cli/wp-cli/issues/4880

Comment (swissspidy) [reply src/MakePotCommand.php]: Only now saw this. The same applies to the language command as well, so 👍

**Pull request merged**